### PR TITLE
Fix PEP-8 violations reported by flake8 on asv/main.py

### PR DESCRIPTION
**Pull request by LucyJimenez**
Remove libraries imported but unused.

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
Thanks @LucyJimenez 

PRs look great. We'll wait until we have flake8 running the in the CI before merging, so we can see that the changes look ok also in the CI. But they look great, thanks for the fixes.

**Comment by LucyJimenez** [comment]
Superseded by #988

**Pull request closed without merging**
